MaybeGame/GameMaster.cs

```diff
@@ -3,69 +3,92 @@
+                        "3. Dump all player information\n" +
+                        "4. Dump all enemy information\n" +
+                        "5. Quit\n");
 
-            public void Start()
+            var success = int.TryParse(Console.ReadLine(), out var menu);
+            if (success == false)
             {
+                Console.WriteLine("You have entered a wrong number");
                 WriteGameMenu();
             }
 
-            private void WriteGameMenu()
+            var playerWantsToQuit = false;
+            switch (menu)
             {
-                Console.WriteLine("\nMenu:\n" +
-                            "1. Add new player\n" +
-                            "2. Dump all information\n" +
-                            "3. Quit" +
-                            "4. LevelUp");
-
-                var success = int.TryParse(Console.ReadLine(), out var menu);
-                if (success == false)
-                {
-                    Console.WriteLine("You have entered a wrong number");
-                    WriteGameMenu();
-                }
+                case 1:
+                    AddNewPlayer();
+                    break;
+                case 2:
+                    AddEnemies();
+                    break;
+                case 3:
+                    DumpPlayersData();
+                    break;
+                case 4:
+                    DumpEnemiesData();
+                    break;
+                case 5:
+                    playerWantsToQuit = true;
+                    break;
+                default:
+                    Console.WriteLine("I am sorry. but this doesn't work");
+                    break;
+            }
 
-                var playerWantsToQuit = false;
-                switch (menu)
-                {
-                    case 1:
-                        AddNewPlayer();
-                        break;
-                    case 2:
-                        DumpPlayersData();
-                        break;
-                    case 3:
-                        playerWantsToQuit = true;
-                        break;
-                    default:
-                        Console.WriteLine("I am sorry. but this doesn't work");
-                        break;
-                }
+            if (playerWantsToQuit) return;
+            else Start();
+        }
 
-                if (playerWantsToQuit) return;
-                else Start();
-            }
+        private void AddNewPlayer()
+        {
+            Console.WriteLine("Enter player name");
+            var userName = Console.ReadLine();
+            Console.WriteLine("Enter clan name");
+            var clanName = Console.ReadLine();
+            var player = new Player();
+            player.StatSet(userName, clanName);
+            _players.Add(player);
+        }
+        private void AddEnemies()
+        {
+            Console.WriteLine("Adding enemies");
+            _enemies.Add(EnemyConfig.CreateZombie());
```

Comment:
Perhaps you could introduce a method **GetDefaultEnemies()** in EnemyConfig and return this list from there instead. Then add it with  _enemies.AddRange() method

Reply:
Will try to do that